2022-07_sprint/sprint_parties/media_kit.md

```diff
@@ -0,0 +1,56 @@
+# Social Media Kit
+
+(media_kit)=
+
+## Event Hashtag
+
+#DataUmbrellaPyMCSprint
```

Comment:
yes, let's keep it. It's already on the social media cards.  And it's short, so easy to remember. Of course, knowing it's not totally consistent. It would be nice if more folks used it, but people tend to forget anyway.